cmd/influxd/server_integration_test.go

```diff
@@ -177,7 +178,7 @@ func write(t *testing.T, node *Node, data string) {
 
 // query executes the given query against all nodes in the cluster, and verifies no errors occured, and
 // ensures the returned data is as expected
-func query(t *testing.T, nodes Cluster, urlDb, query, expected string) (string, bool) {
+func query(t *testing.T, nodes Cluster, urlDb, query, expected, expectPattern string) (string, bool) {
```

Comment:
Perhaps we should have two  versions of this function `query` and `queryRegex`. You could factor out the common code, but two functions might be cleaner.